Gems/Atom/RHI/Code/Include/Atom/RHI/FrameGraphExecuteGroup.h

```diff
@@ -98,10 +101,16 @@ namespace AZ
                 /// user wishes to fill in the command list at context creation time.
                 CommandList* m_commandList = nullptr;
 
-                /// An ordered list of scope ids in the group.
-                const ScopeId* m_scopeIds = nullptr;
+                /// An ordered list of scope ids and submit counts in the group.
```

Comment:
the ScopeEntry struct does not contain a list. Is this a stale comment?

Reply:
Ahh ok. Maybe remove the extra line between ScopeEntry and m_scopeEntries just to make it obvious.

Reply:
Comment moved.